docs/blog/newsletter-october-2021.rst

```diff
@@ -0,0 +1,92 @@
+
+.. post:: October 05, 2021
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – October 2021
+#########################################
+
+Hello everyone!
+
+It's Beth here, writing while I'm listening in to Write the Docs Prague. I hope everyone who attended had a great time! For those of you who couldn't make it, keep an eye out: we'll be publishing the talk videos on YouTube fairly soon. And if you'd like to get more of a flavour of the conference, you can listen to `the intermission music <https://open.spotify.com/artist/36ZvEAnAbW8NOJK7TrBoh5>`__ - which I am bopping along to as I type :D
```

Comment:
btw - I liked the idea of people listening to the same music I was while they read, but let me know if you think it's weird!

Reply:
That's a really awesome thought! I'm just not sure if Spotify is the most accessible platform for some people... Maybe you can reword to invite people to visit my artist website so they can decide how to consume the music?

Reply:
Very cool idea!

Reply:
Sorry, I forget as I use spotify - will switch up the link!